perllib/FixMyStreet/Template.pm

```diff
@@ -175,6 +175,7 @@ sub email_sanitize_text : Fn('email_sanitize_text') {
 
     my $text = $update->{item_text};
     my $extra = $update->{item_extra};
+    utf8::encode($extra);
```

Comment:
This (and L238) causes a `Use of uninitialized value in subroutine entry` warning when running tests in `alert_new.t` for some reason.

Reply:
Will add `if $extra` to end.